### PR TITLE
Fix Russian

### PR DESCRIPTION
**Pull request by greenvorbisfan**

### CODE REVIEW

**Comment by SoongNoonien** [comment]
Thanks! It seems like you accidentally modified the `.gitignore` along the way. Can you fix this?

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 /builddir/
 /repo/
 /.flatpak-builder/
+update
```
Comment (SoongNoonien) [inline .gitignore:4]: I meant this and the following now two lines.

Comment (SoongNoonien) [reply .gitignore:4]: If you like I can fix it as well.

**Comment by SoongNoonien** [comment]
Another quick note: You have configured a different email address locally than in your GitHub account. This has the possibly unwanted effect that the commits you make aren't liked to your GitHub account.

File: .gitignore (hunk 1 above)
Comment (greenvorbisfan) [reply .gitignore]: Fixed.

**Comment by greenvorbisfan** [comment]
Also fixed. :)

**Comment by SoongNoonien** [comment]
Nice! Another small cosmetic thing. Currently there are 3 separate commits in this PR. I like to use a rebase merge with just one commit. So can you combine your three commits into one? Then it's ready for merging.

**Comment by SoongNoonien** [comment]
If you're having trouble with that I can give you guidance or do it myself. Just say what you like best.

**Comment by greenvorbisfan** [comment]
> Just say what you like best.

If it's not difficult for you, do it yourself, please. I'm still learning how to use git. :)

**Comment by SoongNoonien** [comment]
> I'm still learning how to use git. :)

Git is not so easy to learn. Here is what I did:

```
$ git reset HEAD~2
$ git commit --all --amend
$ git push --force
```

**Pull request merged**